src/dmd/nogc.d

```diff
@@ -215,7 +215,10 @@ public:
 extern (C++) Expression checkGC(Scope* sc, Expression e)
 {
     FuncDeclaration f = sc.func;
-    if (e && e.op != TOK.error && f && sc.intypeof != 1 && !(sc.flags & SCOPE.ctfe) && (f.type.ty == Tfunction && (cast(TypeFunction)f.type).isnogc || (f.flags & FUNCFLAG.nogcInprocess) || global.params.vgc))
+    if (e && e.op != TOK.error && f && sc.intypeof != 1 && !(sc.flags & SCOPE.ctfe)
+            && (f.type.ty == Tfunction
+            && (cast(TypeFunction)f.type).isnogc || (f.flags & FUNCFLAG.nogcInprocess) || global.params.vgc)
+            && !(sc.flags & SCOPE.debug_))
```

Comment:
The code is good, but you're inventing another style of doing multiline conditionals.
```
if (e && e.op != TOK.error && f && sc.intypeof != 1 && !(sc.flags & SCOPE.ctfe) &&
     (f.type.ty == Tfunction &&
      (cast(TypeFunction)f.type).isnogc || (f.flags & FUNCFLAG.nogcInprocess) || global.params.vgc) &&
     !(sc.flags & SCOPE.debug_))
```
I.e. the dangling operator goes at the end, not the beginning, and subsequent lines should line up with the character after the opening `(`.